pontoon/base/utils.py

```diff
@@ -9,6 +9,10 @@
 import tempfile
 import time
 import zipfile
+from xml.sax.saxutils import (
+    escape as xml_escape,
+    quoteattr,
+)
```

Comment:
Let's move this block where it belongs alphabetically - under `from translate.lang import data as lang_data`.